FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
```

Comment:
No need to used `self` inside the block, thus no need to unwrap it. It is enough just to check if it is not nil.

Please replace conditional unwrapping with check for `nil`

```suggestion
			guard self != nil else { return }
```